com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/archive/ArchiveBuilder.java

```diff
@@ -371,6 +371,16 @@ public static void main(String[] args) throws IOException, NoSuchAlgorithmExcept
                     printUsageAndTerminate("file does not exist: " + currentInput.getAbsolutePath());
                 }
 
+                if (args[i].endsWith("fontc")) {
+                    File glyphBankFile = new File(args[i].replace("fontc", "glyph_bankc"));
+
+                    if (!glyphBankFile.isFile()) {
+                        printUsageAndTerminate("file does not exist: " + glyphBankFile.getAbsolutePath());
+                    }
+
+                    inputs.add(glyphBankFile);
+                }
```

Comment:
Not sure if there's a better way to solve this.. we need to be able to include source files that generate more than one build target, which we currently don't support.